enterprisemanagementv1/enterprise_management_v1_integration_test.go

```diff
@@ -48,18 +48,20 @@ var _ = Describe(`EnterpriseManagementV1 Integration Tests`, func() {
 		serviceURL                  string
 		testConfig                  map[string]string
 
-		enterpriseId                        string
-		accountId                           string
-		accountIamId                        string
-		firstExampleAccountGroupName        = "Example Account Group"
-		firstUpdatedExampleAccountGroupName = "Updated Example Account Group"
-		secondExampleAccountGroupName       = "Second Example Account Group"
-		exampleAccountName                  = "Example Account Name"
-		exampleAccountId                    *string
-		resultPerPage                       int64 = 1
-		firstExampleAccountGroupId          *string
-		secondExampleAccountGroupId         *string
-		updatedEnterpriseName               = "Updated Enterprise Name"
+		enterpriseId string
+		accountId    string
+		accountIamId string
+
+		accountGroupName          = "Example Account Group"
+		accountGroupId            *string
```

Comment:
You've done a very subtle thing by changing the type of this variable to `*string` and how you use it below.
It is now a pointer and so you are actually saving the ADDRESS OF the `AccountGroupID` field within the response struct.   This is risky IMO.  I think it's better to use non-pointer types for the global variables that are used to hold values that link together operations.